framework/include/cppmicroservices/AnyMap.h

```diff
@@ -330,8 +330,43 @@ class US_Framework_EXPORT AnyMap : public any_map
    * @throws std::invalid_argument if the \c Any value for a given key is not of type \c AnyMap or \c std::vector<Any>.
    * @throws std::out_of_range if the key is not found or a numerical index would fall out of the range of an \c int type.
    */
-  mapped_type& AtCompoundKey(const key_type& key);
```

Comment:
The implementation of this function simply called the const version and applied a const_cast which seemed odd, so I decided to remove it. It may be backwards incompatible in the technical sense but I felt comfortable removing it because there are no users for it. I can put it back if needed.